### ISSUE TITLE
Elements under modal dialogs remain visible to screen readers

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6075]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.770
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Start a screen reader (using VoiceOver on mac for this example)
- Bring up a modal dialog in RStudio, e.g. Help / About RStudio
- Try to move the screen-reader cursor to elements outside the modal dialog

### Describe the problem in detail
- With VoiceOver, you can VO+Shift+Up to get out of the dialog, and then VO+Left/Right to move between elements in the main UI (under the "glass").

### Describe the behavior you expected
- In addition to being inaccessible via Tab key, elements under a modal dialog should also be invisible to screen readers

### Other Problems
- When a modal dialog invokes another modal dialog (stacked dialogs), the original dialog remains visible to screen readers and to the tab sequence (i.e. covered dialogs need to be made inert, too)
- Satellite Windows also need to be considered; when they show modal dialogs the satellite window's main contents should be made inert

**Comment by gtritchie** [comment]
https://github.com/GoogleChrome/inert-polyfill/issues/15

**Comment by rich-rstudio** [comment]
@gtritchie On RStudio Server 1.3.888, Safari, MacOS 10.14.6 I can shift focus to the console with Shift + arrow keys when the "About RStudio"  modal is still up and the VoiceOver will read out the console output

**Comment by gtritchie** [comment]
That's a known bug in VoiceOver. Been there for years. Using Arrow keys (including Shift+arrow keys) is not a supported valid way to move VoiceOver cursor (even though it does move around, but ignores the aria-hidden attribute).

Move VoiceOver cursor using VO+Shift+Arrow keys (where VO depends how you have VoiceOver configured; by default it is Ctrl+Alt, but recommend changing it to be CapLock).

**Comment by rich-rstudio** [comment]
i see thanks @gtritchie

verified in 1.3888
